cmd/fastly/main.go

```diff
@@ -88,7 +87,7 @@ func main() {
 	// Extract a subset of configuration options from the local application directory.
 	var file config.File
 	file.SetStatic(cfg)
-	err = file.Read(config.FilePath, in, out)
+	err = file.Read(config.FilePath, in, out, fsterr.Log)
```

Comment:
Updated the method signature so that we can internally record any errors to our `errors.log` file on disk.